src/site/content/en/blog/webcodecs/index.md

```diff
@@ -145,46 +159,51 @@ let config = {
   codec: 'vp8',
   width: 640,
   height: 480,
-  bitrate: 8_000_000,     // 8 Mbps
+  bitrate: 2_000_000, // 2 Mbps
   framerate: 30,
 };
 
 let encoder = new VideoEncoder(init);
 encoder.configure(config);
 ```
 
-After the encoder has been set up,  it's ready to start accepting frames. When
-frames are coming from a media stream, the callback given to `VideoTrackReader.start()` will pump frames into
-the encoder, periodically inserting
-[keyframes](https://en.wikipedia.org/wiki/Key_frame#Video_compression) and
-checking that the encoder is not overwhelmed with incoming frames.
+After the encoder has been set up, it's ready to start accepting frames via `encode()` method.
 Both `configure()` and `encode()` return immediately without waiting for the
 actual work to complete. It allows several frames to queue for encoding at the
-same time. But it makes error reporting somewhat cumbersome. Errors are reported
-either by immediately throwing exceptions or by calling the `error()`
-callback. Some errors are easy to detect immediately, others become evident
-only during encoding. If encoding completes successfully the `output()`
+same time, `encodeQueueSize` shows how many requests are waiting in the queue waiting
```

Comment:
```suggestion
same time, while `encodeQueueSize` shows how many requests are waiting in the queue
```

Reply:
The alternative to "while" is to make `encodeQueueSize` the start of another sentence.